boards/arm/nucleo_l476rg/Makefile.board

```diff
@@ -0,0 +1,7 @@
+FLASH_SCRIPT = openocd.sh
+DEBUG_SCRIPT = openocd.sh
+
+OPENOCD_LOAD_CMD = "flash write_image erase ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
+OPENOCD_VERIFY_CMD = "verify_image ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
+
+export FLASH_SCRIPT OPENOCD_LOAD_CMD OPENOCD_VERIFY_CMD
```

Comment:
This needs to be ported to CMake.

Doing that involves renaming this file to board.cmake and replacing it's contents with

"include($ENV{ZEPHYR_BASE}/boards/common/openocd.board.cmake)"

The same with the other board in this PR.

Reply:
Alright I've done this and tested for both boards, this works on my end.